client/main.lua

```diff
@@ -458,7 +458,7 @@ Citizen.CreateThread(function()
                                         if QBCore.Functions.GetPlayerData().charinfo.gender == 1 then gender = "female" end
                                         QBCore.Functions.TriggerCallback('qb-clothing:server:getOutfits', function(result)
                                             openMenu({
-                                                {menu = "roomOutfits", label = "Presets", selected = true, outfits = Config.Outfits[PlayerData.gang.name][gender]},
+                                                {menu = "roomOutfits", label = "Presets", selected = true, outfits = Config.Outfits[PlayerData.gang.name][gender][PlayerData.job.grade.level]},
```

Comment:
This shouldn't use `PlayerData.job.grade.level` since its a gang it should be `PlayerData.gang.grade.level`

Reply:
Thanks.  Will Fix In Second Update.  Standby